### DISCUSSION TITLE
Lista fatture cliente

### DISCUSSION THREAD

**Discussion by francescoc100** [fattureincloud/api#276] [Q&A] [answered]
Ciao, sto facendo alcune prove oggi usando  List Issued Documents, tuttavia ho la neccessità di filtrare i risultati per cliente. Prima utilizzavo la query sul nome, se provo a fare nello stesso modo leggo:
"message": "Cannot filter by \"name\" field.",

Non si può più fare?

**Comment by francescoc100** [comment, marked as answer]
Risolto usando entity.name='nome-cliente'

**Reply by valmoz** [reply]
Bene @francescoc100 
sono felice che sia riuscito a risolvere da solo.

La lista dei campi su cui è possibile basare i filtri si trova qui: [Filterable Fields](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results#-filterable-fields)

A disposizione
Mauro
